### PR TITLE
fix(components): fix errors on proptypes

### PR DESCRIPTION
**Pull request by jare-talend**
**What is the problem this PR is trying to solve?**

fix errors in the console when we use ResourcePicker

**What is the chosen solution to this problem?**

Fix the proptype defintion 

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/RichTooltip/RichLayout/RichLayout.component.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import Inject from '../../Inject';
 import theme from './RichLayout.scss';
 
 const TooltipPropTypes = {
-	id: PropTypes.string.isRequired,
+	id: PropTypes.string,
```
Comment (jsomsanith-tlnd) [inline packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: So you will have `undefined-content` as id sometimes `id={`${id}-content`}`

Comment (jare-talend) [reply packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: For Body yes but Header and Footer no
And using class undefined-content is dirty. Id should be uniq

File: packages/components/src/RichTooltip/RichLayout/RichLayout.component.js [hunk 2]
```diff
@@ -60,4 +64,5 @@ RichLayout.propTypes = {
 	Header: Inject.getReactElement.propTypes,
 	Footer: Inject.getReactElement.propTypes,
 	text: PropTypes.string,
+	prefixId: PropTypes.string.isRequired,
```
Comment (frassinier) [inline packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: WDYT? to be consistent!
```suggestion
	id: PropTypes.string.isRequired,
```

Comment (jare-talend) [reply packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: I expect that if i name the variable "id", i do id={id} in the component. In my case, I use a prefix to create the id used by the component, so, why i name it prefixId. However, i don't disturb me to change to id if you prefer.

Comment (jsomsanith-tlnd) [reply packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: If it bothers you, you can implement the behavior you just described, adding the id on the main element wrapper, and still use this id to generate ids on sub parts

Comment (frassinier) [reply packages/components/src/RichTooltip/RichLayout/RichLayout.component.js]: +1 yes it sounds great

**Pull request merged**